docs/processing/preprocessing.md

```diff
@@ -1,7 +1,8 @@
 ## Executing *fMRIPrep*
 Because *fMRIPrep* creates a single anatomical reference for all sessions, we generate such reference first by setting the `--anat-only` flag.
 If that *fMRIPrep* execution finishes successfully, the anatomical processing outcomes will be stored in the output folder.
-We will then run one *fMRIPrep* process for each dataset's session, which is the recommended way for datasets with a large number of sessions (e.g., more than six sessions).
+We will then run one *fMRIPrep* process for each dataset's session, which is the recommended way for datasets with a large number of sessions (e.g., more than 
+six sessions).
```

Comment:
Why this line break?

```suggestion
We will then run one *fMRIPrep* process for each dataset's session, which is the recommended way for datasets with a large number of sessions (e.g., more than  six sessions).
```

Reply:
I'm not sure why I have many seemingly random line break ?!